src/parser.jl

```diff
@@ -1,5 +1,5 @@
 if isdefined(Base, :isnumeric)
-    _isalnum(c) = isalpha(c) || isnumeric(c)
+    _isalnum(c) = isletter(c) || isnumeric(c)
```

Comment:
Since `isnumeric` exists in the minimum required version of Julia here, we should be able to remove this conditional entirely and simply replace calls to `_isalnum` with the definition. Or if that gets too verbose, we could keep `_isalnum` and simply drop the conditional.